Fatras/include/ActsFatras/Digitization/PlanarSurfaceDrift.hpp

```diff
@@ -0,0 +1,49 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Definitions/Algebra.hpp"
+#include "Acts/Geometry/GeometryContext.hpp"
+#include "Acts/Surfaces/Surface.hpp"
+#include "Acts/Surfaces/SurfaceError.hpp"
+#include "Acts/Utilities/Helpers.hpp"
+
+#include <functional>
+
+namespace ActsFatras {
+
+/// The PlanarSurfaceDrift takes an intersection in the nominal surface and
+/// projects the ends into the readout surface, which can be at : -1, 0, 1
+///
+/// A Lorentz drift angle can be applied, such as a small smearing to it
```

Comment:
I think a work is missing ? Maybe it should be "such as **to add** a small smearing to it" or maybe I am missing the meaning of the sentence.

Reply:
Ah, I changed that, I leave it to the caller to smear before drifting.

Reply:
Changed.